.env.example

```
@@ -1,64 +1,7 @@
-NODE_PATH=
-REACT_APP_CHAIN_ID=3
-REACT_APP_PEER_URL=https://peer.decentraland.org
-REACT_APP_ERC721_COLLECTION_BASE_URI=https://peer.decentraland.org/lambdas/collections/standard/erc721/
-REACT_APP_LAND_MANAGER_URL=https://api.thegraph.com/subgraphs/name/decentraland/land-manager
-REACT_APP_EMAIL_SERVER_URL=
-REACT_APP_MARKETPLACE_URL=
-REACT_APP_MARKETPLACE_WEB_URL=
-REACT_APP_BUILDER_SERVER_URL=
-REACT_APP_ENVIRONMENT=development
-
-# Auth
-REACT_APP_AUTH0_CLIENT_ID=
-REACT_APP_AUTH0_DOMAIN=
-REACT_APP_AUTH0_REDIRECT=
-REACT_APP_AUTH0_AUDIENCE=
-
-# Crypto
-
-REACT_APP_IDENTITY_EXPIRATION_MINUTES=
-REACT_APP_MIN_SALE_VALUE_IN_WEI=
-
-# Third party
-
-REACT_APP_SEGMENT_API_KEY=
-REACT_APP_HOTJAR_ID=
-REACT_APP_INTERCOM_APP_ID=
-REACT_APP_ROLLBAR_ACCESS_TOKEN=
-REACT_APP_DELIGHTED_API_KEY=
-REACT_APP_EXPLORER_URL=https://play.decentraland.org
-REACT_APP_TRANSACTIONS_API_URL=https://transactions-api.decentraland.co/v1
-REACT_APP_MARKETPLACE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/marketplace
-REACT_APP_IPFS_URL=https://ipfs.infura.io:5001/api/v0/add?pin=false
-REACT_APP_ACCOUNT_URL=https://account.decentraland.org
-REACT_APP_DISCORD_URL=https://dcl.gg/discord
-REACT_APP_WEARABLES_ZIP_INFRA_URL=http://a-test-url-com
-
-# Contracts
-
-REACT_APP_MANA_TOKEN_CONTRACT_ADDRESS=0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb
-REACT_APP_LAND_REGISTRY_CONTRACT_ADDRESS=0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d
-REACT_APP_ESTATE_REGISTRY_CONTRACT_ADDRESS=0x959e104e1a4db6317fa58f8295f586e1a978c297
-REACT_APP_ERC721_COLLECTION_FACTORY_CONTRACT_ADDRESS=
-REACT_APP_ERC721_COLLECTION_CONTRACT_ADDRESS=
-REACT_APP_ENS_CONTRACT_ADDRESS=0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e
-REACT_APP_RESOLVER_CONTRACT_ADDRESS=0x4976fb03c32e5b8cfe2b6ccb31c09ba78ebaba41
-REACT_APP_CONTROLLER_ADDRESS=0xd4e92f6ea18412dbae8b2d2b69e498a2d1064fa2
-REACT_APP_REGISTRAR_ADDRESS=0xeb6f5d94d79f0750781cc962908b161b95192f53
-
-# Others
-
-REACT_APP_PROMO_URL=
-REACT_APP_SHARE_SCENE_URL=
+NODE_PATH=development
```

Shouldn't `NODE_PATH` be src?
```suggestion
NODE_PATH=src
```
Shouldn't it be in the other environment files?

---

yes 😅